addons/mute-project/userscript.js

```diff
@@ -4,12 +4,26 @@ export default async function ({ addon, global, console }) {
   let icon = document.createElement("img");
   icon.src = "/static/assets/e21225ab4b675bc61eed30cfb510c288.svg";
   icon.style.display = "none";
+  let ctrlPressesCount = 0;
+  let ctrlPressedRecently = false;
+
+  window.addEventListener("keydown", (event) => {
+    if (event.ctrlKey) {
+      ctrlPressesCount++;
+      const pressCount = ctrlPressesCount;
+      ctrlPressedRecently = true;
+      setTimeout(() => {
+        if (pressCount === ctrlPressesCount) ctrlPressedRecently = false;
+      }, 2500);
```

Comment:
We might want to lower this to less than 2.5 seconds, but in that case Alt+GreenFlag for 60FPS should also be updated, so better to do in a separate PR if someone is interested.